app/src/main/java/com/edulectronics/tinycircuit/ui/CircuitActivity.java

```diff
@@ -32,8 +36,23 @@ public boolean onTouch(View v, MotionEvent event) {
             }
         });
         Intent intent = getIntent();
-        CircuitController controller = (CircuitController) intent.getSerializableExtra("Controller");
+        CircuitController controller =
+                (CircuitController) intent.getSerializableExtra("Controller");
         circuit.setNumColumns(controller.circuit.width);
         circuit.setAdapter(new CircuitAdapter(this, controller));
+
+        /*This code adds a menu to the side*/
+        ListView menu = (ListView) findViewById(R.id.menu);
+        String[] items = {"Item1", "Item2"};
```

Comment:
Hoewel dit nog veranderd moet worden, zou ik deze string array toch even in de strings.xml file zetten. (zie https://developer.android.com/guide/topics/resources/string-resource.html#StringArray)